website/siteConfig.js

```diff
@@ -14,12 +14,84 @@ const baseUrl = process.env.BASE_URL || (process.env.CONTEXT ? '/' : '/KaTeX/');
 
 /* List of projects/orgs using your project for the users page */
 const users = [
+    {
+        caption: 'Bokeh',
+        image: 'https://bokeh.pydata.org/en/latest/_static/images/logo.png',
+        infoLink: 'https://bokeh.pydata.org/',
+        pinned: true,
+    },
+    {
+        caption: 'CindyJS',
+        image: 'https://cindyjs.org/assets/img/logo.png',
+        infoLink: 'https://cindyjs.org/',
+        pinned: true,
+    },
+    {
+        caption: 'Dropbox Paper',
+        image: 'https://aem.dropbox.com/cms/content/dam/dropbox/www/en-us/branding/app-paper-ios@2x.png',
+        infoLink: 'https://paper.dropbox.com/',
+        pinned: true,
+    },
+    {
+        caption: 'Expii',
+        image: 'https://78.media.tumblr.com/avatar_ccde3e53e98f_128.pnj',
+        infoLink: 'https://www.expii.com/',
+        pinned: true,
+    },
+    {
+        caption: 'Facebook Messenger',
+        image: 'https://en.facebookbrand.com/wp-content/uploads/2016/09/messenger_icon2.png',
+        infoLink: 'https://www.messenger.com/',
+        pinned: true,
+    },
     {
         caption: 'GitLab',
         image: 'https://gitlab.com/gitlab-com/gitlab-artwork/raw/master/logo/logo-square.png',
         infoLink: 'https://gitlab.com/',
         pinned: true,
     },
+    {
+        caption: 'Gitter',
+        image: 'https://assets.gitlab-static.net/uploads/-/system/project/avatar/3601513/gitter_logo.png',
+        infoLink: 'https://gitter.im/',
+        pinned: true,
+    },
+    {
+        caption: 'Gradescope',
+        image: 'https://pbs.twimg.com/profile_images/920085750095298560/AqGnlpiJ_400x400.jpg',
+        infoLink: 'https://www.gradescope.com/',
+        pinned: true,
+    },
+    {
+        caption: 'hack.chat',
+        image: 'https://hack.chat/apple-icon-180x180.png',
+        infoLink: 'https://hack.chat/',
+        pinned: true,
+    },
+    {
+        caption: 'Khan Academy',
```

Comment:
I think Khan Academy should be on a separate section or the first, as this project is maintained by them.

Reply:
OK, maybe KA first and the rest alphabetical (i.e. as is)?